basyx.submodelrepository/basyx.submodelrepository-backend-inmemory/src/main/java/org/eclipse/digitaltwin/basyx/submodelrepository/InMemorySubmodelRepository.java

```diff
@@ -173,9 +174,11 @@ public void updateSubmodel(String id, Submodel submodel) throws ElementDoesNotEx
 	}
 
 	@Override
-	public void createSubmodel(Submodel submodel) throws CollidingIdentifierException {
+	public void createSubmodel(Submodel submodel) throws CollidingIdentifierException, MissingIdentifierException {
+		throwIfSubmodelIdEmptyOrNull(submodel.getId());
 		throwIfSubmodelExists(submodel.getId());
```

Comment:
Please consider adding a vertical space above this.

Reply:
done